packages/website/docs/Features/Functions.mdx

```diff
@@ -78,37 +117,141 @@ As well as mixed distributions:
 
 <SquiggleEditor initialSquiggleString="mx(3, 8, 1 to 10, [0.2, 0.3, 0.5])" />
 
-## Other Functions
+An alias of `mx` is `mixture`
```

Comment:
I imagine it would be better to have this the other way; introduce `mixture` first.